### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by 444722146** [vercel/community#3095] [Help] [answered]
### Summary

This user account is blocked.

### Example

This user account is blocked.

### Steps to Reproduce

This user account is blocked.

**Comment by 444722146** [comment]
My account cannot be logged in. How can I log in normally? There are very important things inside!

**Reply by amyegan** [reply, marked as answer]
Please [submit a case](https://vercel.com/help#issues) to the Customer Success Team for help logging into your blocked account 🙏

**Comment by 444722146** [comment]
***@***.***，There is very important information, can you help me unblock it? I am willing to pay
&nbsp;


吾爱学习
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年7月3日(星期一) 晚上9:44
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [vercel/community] This user account is blocked. (Discussion #3095)






请提交案例向Customer Success Team寻求帮助，登录您被冻结的帐户🙏

—
直接回复此邮件，在GitHub上查看，或者取消订阅.
***@***.***与&gt;.
